daml-assistant/src/DAML/Assistant/Install.hs

```diff
@@ -173,10 +184,44 @@ activateDaml env@InstallEnv{..} targetPath = do
                      ]
             else createSymbolicLink damlBinarySourcePath damlBinaryTargetPath
 
-    unlessQuiet env $ do -- Ask user to add .daml/bin to PATH if it is absent.
-        searchPaths <- map dropTrailingPathSeparator <$> getSearchPath
-        when (damlBinaryTargetDir `notElem` searchPaths) $ do
-            output ("Please add " <> damlBinaryTargetDir <> " to your PATH.")
+    updatePath env damlBinaryTargetDir
+
+updatePath :: InstallEnv -> FilePath -> IO ()
+#ifdef mingw32_HOST_OS
```

Comment:
I would be happier if updatePath just had an `| isWindows` check as we do elsewhere that called a separate module (`DAML.Assistant.Install.Win32`) with all the conditional compilation.

Reply:
I’ll move it to `DAML.Assistant.Install.Path`. Having a module called `Win32` that also contains the logic used outside of Windows seems weird.